tests/modules/test_nat.py

```diff
@@ -1,14 +1,12 @@
 import unittest
-import tempfile
-import yaml
 from genisys.modules.nat import Nat
 from genisys import configParser
 
 class NatTest(unittest.TestCase):
     """ Tests for the NAT module """
     def test_values_in_config(self):
         """ Ensure that a missing config option raises the expected error. """
-        with open("../configs/nat_test_1.yml") as config_file:
+        with open("../genisys/tests/configs/nat_test_1.yml") as config_file:
```

Comment:
"genisys" is the repo root. If you're running the test from the root directory of the repo, this is saying to open `genisys/../genisys/tests/configs/net_test_1.yml` which isn't technically wrong but just redundant. It should just be `./tests/configs/nat_test_1.yml` or `tests/configs/nat_test_1.yml` since the `./` is implied.

Reply:
Done